### ISSUE TITLE
`flet pack` command doesn't include `web` folder into `flet` folder of standalone executable

### ISSUE THREAD

**Issue by danil09234** [flet-dev/flet#1567]
**Description**

Hello! When I'm trying to run Flet app package created using `flet pack main.py --add-data "fonts;fonts"` command, I'm getting the next error message window:

```
Traceback (most recent call last):
  File "main.py", line 466, in <module>
  File "flet_runtime\app.py", line 81, in app
  File "asyncio\base_events.py", line 650, in run_until_complete
  File "flet_runtime\app.py", line 223, in app_async
  File "flet_runtime\app.py", line 416, in __connect_internal_async
  File "flet_runtime\app.py", line 537, in __start_flet_server
Exception: Web root path not found: C:\Users\user\AppData\Local\Temp\_MEI260322\flet\web
```

**Code example to reproduce the issue:**

```python
import flet


async def main(page: flet.Page):
    """
    My app code here
    """
    pass


flet.app(target=main, view=flet.AppView.WEB_BROWSER)
```

**Describe the results you received:**

Error window of an unhandled exception with the message above. 

**Describe the results you expected:**

I expect the `flet pack` to simply create a functioning standalone web application : ) 

**Additional information you deem important (e.g. issue happens only occasionally):**

When I packed my application with `flet pack main.py -D --add-data "fonts;fonts"` and added the folder `venv/Lib/site-packages/flet/web` to the `dist/main/flet` folder, my application successfully deployed)

**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.8.1
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page:
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: D:\MyWorkingDirHere\venv\Lib\site-packages
Requires: copier, flet-runtime, packaging, pydantic, qrcode, watchdog, websocket-client, websockets
Required-by:
```

**PyInstaller version (`pip show pyinstaller`)**:
```
Name: pyinstaller
Version: 5.13.0
Summary: PyInstaller bundles a Python application and all its dependencies into a single package.
Home-page: https://www.pyinstaller.org/
Author: Hartmut Goebel, Giovanni Bajo, David Vierra, David Cortesi, Martin Zibricky
Author-email:
License: GPLv2-or-later with a special exception which allows to use PyInstaller to build and distribute non-free programs (including commercial ones)
Location: D:\MyWorkingDirHere\venv\Lib\site-packages
Requires: altgraph, pefile, pyinstaller-hooks-contrib, pywin32-ctypes, setuptools
Required-by:
```

**Operating system**:
Windows 10

**Additional environment details:**
I am using a venv created by PyCharm Community edition.

**Comment by danil09234** [comment]
Additionally, modifying `main.spec` after `flet pack main.py --add-data "fonts;fonts"` and running `pyinstaller main.spec` helps to create functioning standalone `.exe`:

**`main.spec` after modification:**
```
...
a = Analysis(
    datas=[('fonts', 'fonts'), ('venv/Lib/site-packages/flet/web', 'flet/web')],
    ...
)
...
```

**Comment by FeodorFitsner** [comment]
Heh, `flet pack` has been made for packaging desktop apps, not web. So your issue is more a request rather than a bug :)

**Comment by danil09234** [comment]
Oh, it seems to me a very useful and important feature. I searched for information about this in the documentation but did not find if it is possible or not. By the way, I only found that even if the application is running in desktop application mode, it is still a web application that has a server running in the background. But how does my solution work stable enough / should it work stable, I don't know something?
